contracts/src/v0.8/KeeperRegistry1_2.sol

```diff
@@ -11,16 +11,16 @@ import "./interfaces/TypeAndVersionInterface.sol";
 import "./interfaces/AggregatorV3Interface.sol";
 import "./interfaces/LinkTokenInterface.sol";
 import "./interfaces/KeeperCompatibleInterface.sol";
-import "./interfaces/KeeperRegistryInterface.sol";
-import "./interfaces/MigratableKeeperRegistryInterface.sol";
-import "./interfaces/UpkeepTranscoderInterface.sol";
+import "./interfaces/KeeperRegistryInterface1_2.sol";
+import "./interfaces/MigratableKeeperRegistryInterface1_2.sol";
+import "./interfaces/UpkeepTranscoderInterface1_2.sol";
```

Comment:
I think for these two we can keep the same for now? Don't think we envision features changing these interfaces?

Reply:
We will have a new UpkeepFormat (from V1 to both V1 & V2). Maybe we can just modify the existing UpkeepFormat enum sol file and not changing these.

Reply:
I think it's fine to keep a single upkeep format file across different versions as we just append to enum